packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -247,7 +293,46 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
   }
 
   private _onScrollEvent = (container: HTMLElement, footerStickyContainer: HTMLElement): void => {
-    if (this.root && this.nonStickyContent) {
+    const { scrollablePane } = this.context;
+    if (!(this.root && this.nonStickyContent && scrollablePane)) {
+      return;
+    }
+    const { stickyPosition } = this.props;
+    const isHeader = stickyPosition === StickyPositionType.Header;
+    const isFooter = stickyPosition === StickyPositionType.Footer;
+
+    // StickyAlways is not applicable for stickyPosition 'Both'
+    if (
+      isHeader !== isFooter /** position must not be 'Both'*/ &&
+      scrollablePane.verifyStickyContainerBehavior(isHeader, StickyContainerBehaviorType.StickyAlways)
+    ) {
+      // StickyAlways is not applicable for stickyPosition 'Both'
+      // 1. ScrollablePane is mounted and has called notifySubscriber
+      // 2. stickyAlways has to re-render if mutation could 've affected it's offsetHeight.
+
+      // stickyAlways doesn't apply to StickyPosition.Both
+      this.setState({
+        isStickyTop: isHeader && true,
+        isStickyBottom: !isHeader && true,
```

Comment:
The above boolean conditionals are confusing and seem unnecessary. Can they be removed?

```
false && true
> false
true && true
> true
```

If you need to coerce the values to `boolean` then we can find a more readable way.

Reply:
will clean this up